resources/js/App/Components/LayoutRight/LayoutRight.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./layoutright.css";
+
+import Introduction from "./Introduction/Introduction.jsx";
+import VideoPage from "./VideoPage/Videopage.jsx";
+import { Switch, Route } from "react-router-dom";
+
+export default class LayoutRight extends React.Component {
+  render() {
+    return(
+      <div className="layoutright">
+        <p>Writing from LayoutRight</p>
+        <Switch>
+          <Route exact path="/" component={Introduction} />
+          <Route exact path="/videos/:alias" component={VideoPage} />
```

Comment:
Should probably also add "catch all" 404 page. E.g. if user enters some unknown URL our app will currently display an empty white page. We can leave it for later, for the next sprint.